content/webapp/components/Exhibition/Exhibition.js

```diff
@@ -270,9 +273,13 @@ const Exhibition = ({ exhibition }: Props) => {
             contributors={exhibition.contributors}
           />
         )}
-        {exhibitionOfs.length > 0 && (
-          <SearchResults items={exhibitionOfs} title={`In this exhibition`} />
+        {(exhibitionOfs.length > 0 || pages.length > 0) && (
+          <SearchResults
+            items={[...exhibitionOfs, ...pages]}
```

Comment:
This almost certainly isn't the final resting place of these new bits of related content on the exhibition page, but until we've figured that out, it's possibly the most sensible place for now.

Reply:
Worth leaving the note in the code?